x-pack/plugins/entity_manager/server/lib/entities/built_in/services_from_ecs_data.ts

```diff
@@ -42,8 +42,8 @@ export const builtInServicesFromEcsEntityDefinition: EntityDefinition =
         syncDelay: '2m',
       },
     },
-    identityFields: ['service.name', { field: 'service.environment', optional: true }],
-    displayNameTemplate: '{{service.name}}{{#service.environment}}:{{.}}{{/service.environment}}',
+    identityFields: ['service.name'],
```

Comment:
We should increase the definition version. an update identityFields should be a major change imo

Reply:
https://github.com/elastic/kibana/pull/194451/commits/82aa40df1d29bfa79e27b98a3124a358fbfbcae4

Reply:
Correct I meant major version bump. note that `service.environment` is already included in the metadata and my thinking was more about the generated ids. I believe transform won't generate the same id for a given service that previously had name + environment, creating duplicated entities in an index that has version 1 and version 2, while this is implementation detail in kind of break the entities behavior. No strong opinion on this though

Reply:
https://github.com/elastic/kibana/pull/194451/commits/b17af2776af6a31163500b6e9eb95e40867fe369

>Also service.environment can be multi-value now but was previously single value. No strong opinion on this though

We're not using it.

Reply:
I would argue that the version should have started at `0.1.0` because this is pre-GA, `0.2.0` for this change.

Reply:
It sounds good to me. We just need to come to an agreement here.